src/net_processing.cpp

```diff
@@ -3507,6 +3520,83 @@ void PeerManagerImpl::PostProcessMessage(MessageProcessingResult&& result, NodeI
     }
 }
 
+MessageProcessingResult PeerManagerImpl::ProcessPlatformBanMessage(NodeId node, std::string_view msg_type, CDataStream& vRecv)
+{
+    if (msg_type != NetMsgType::PLATFORMBAN) return {};
+
+    // Do nothing if node is out of sync
+    if (!m_mn_sync.IsBlockchainSynced()) {
+        return {};
+    }
+
+    PlatformBanMessage ban_msg;
+    vRecv >> ban_msg;
+
+    const uint256 hash = ban_msg.GetHash();
+
+    LogPrintf("PLATFORMBAN -- hash: %s protx_hash: %s height: %d peer=%d\n", hash.ToString(), ban_msg.m_protx_hash.ToString(), ban_msg.m_requested_height, node);
+
+    WITH_LOCK(::cs_main, EraseObjectRequest(node, CInv{MSG_PLATFORM_BAN, hash}));
+
+    MessageProcessingResult ret{};
```

Comment:
I htink we should let `PostProcessMessage` do the job
```suggestion
    MessageProcessingResult ret{};
    ret.m_to_erase = CInv{MSG_PLATFORM_BAN, hash};

```